shared/redux/rtk-apis/enrollments/enrollments.types.ts

```diff
@@ -3,6 +3,7 @@ export interface IUser {
   firstName: string;
   lastName: string;
   email: string;
+  userType: string;
```

Comment:
Is this needed for user role?

Reply:
Yes bhaiya.

Reply:
In that case, could you specify the roles in the type?